v3/src/components/case-table/collection-table.tsx

```diff
@@ -11,12 +11,16 @@ import { useCollectionContext } from "../../hooks/use-collection-context"
 import { useDataSetContext } from "../../hooks/use-data-set-context"
 import { IDataSet } from "../../models/data/data-set"
 import { useCaseTableModel } from "./use-case-table-model"
-import { CollectionTitle } from './collection-title'
+import { CollectionTitle } from "./collection-title"
 
 import styles from "./case-table-shared.scss"
 import "react-data-grid/lib/styles.css"
 
-export const CollectionTable = observer(function CollectionTable() {
+interface IProps {
+  isTileInFocus: boolean
```

Comment:
Rather than passing this in as a prop, the child can just get it using `useTileModelContext()` which returns the current selection state.